### ISSUE TITLE
Need a little help with the expected structure

### ISSUE THREAD

**Issue by Kimbaras** [Kareadita/Kavita#2919]
### What happened?

Not really a "bug" (I think), but could be used to expand the docs?

I believe it's more on the other side of the monitor :D

### What did you expect?

A little advice / clarification on how the structure folder / parsing is working.

### Kavita Version Number - If you don not see your version number listed, please update Kavita and see if your issue still persists.

0.8.1 - Stable

### What operating system is Kavita being hosted from?

None

### If the issue is being seen on Desktop, what OS are you running where you see the issue?

None

### If the issue is being seen on Mobile, what OS are you running where you see the issue?

None

### Additional Notes

First of all, please excuse me if this is not the best place for such requests (if either prefer this to be a reddit post, on discord or whatever just let me know).

So basically after a couple of days of trying, I need some advice with the expected folder structure as I'm having a little bit of trouble...
For the beginning I'm trying to organize the editions of a tabletop miniature game based on editions (all PDFs files), the current (and wanted) folder structure is:

`
Game (root folder)

    First Edition
        Game - Rulebook.pdf
        Game - Book A.pdf
    Second Edition
        Game - Rulebook.pdf
        Game - Book A.pdf
`

With this structure (and pointing the Kavita library to "Game (root folder)") I'm getting two series, First Edition and Second Edition (correctly).
Inside each series I get all the books under the "Specials" section / tag.
I wanted to the have books recognized as "books" and not specials, so I tried to rename the files as follow

`
Game (root folder)

    First Edition
        Game - Rulebook - vol1.pdf
        Game - Book A - vol2.pdf
    Second Edition
        Game - Rulebook - vol1.pdf
        Game - Book A - vol2.pdf
`

But then I loose the two series and I got a series for each book / file.
I'll add few screenshots (sorry for the wall of text / images):

PS: in the first screen "Prima Edizione" means "First Edition", sorry for the Italian

**First case described**
Game (root folder)
![Root-Folder](https://github.com/Kareadita/Kavita/assets/102742484/41baac1f-c5a5-4321-8177-0bacb9a5e995)

Books / files with no volume number
![books-no-volume](https://github.com/Kareadita/Kavita/assets/102742484/028cead0-420d-48e6-a604-ad6e7ec433c5)

Results in Kavita
![result-1](https://github.com/Kareadita/Kavita/assets/102742484/14ec7bbf-e1d1-4af4-93df-a05b845cd058)
![result-2](https://github.com/Kareadita/Kavita/assets/102742484/5af283a9-65e2-4b30-b28e-b4c0c91c7efe)

**Second case described**
(root folder NOT changed)

Books / files with volume number
![books-volumes](https://github.com/Kareadita/Kavita/assets/102742484/3ca18251-750b-4d35-88b3-ab0d32bbf610)

Results in Kavita
![multiple-series](https://github.com/Kareadita/Kavita/assets/102742484/3697dcbc-e93c-4b2e-96c3-4aa48e1df5a4)
![result-3](https://github.com/Kareadita/Kavita/assets/102742484/b0a29c6e-3783-46f0-ac7a-09f43b48748d)

So yeah, I don't think I've really understood how Kavita is expeting the files to be named or the folder structure...

Also, side question: on the docs (https://wiki.kavitareader.com/guides/scanner/managefiles) there is a mention on additionals keywords for "Specials", this is related to the names of the files or additionals sections that can be displayed on Kavita UI?

As mentioned, not sure if this is the right place to ask such things, so feel free to point me to the correct place and I'll provide!

Thanks!

**Comment by scare376** [comment]
so the problem is that kavita doesnt group things based on folder structure. the folder requirements is there to help the scan loop. PDFs rely rather heavy on filename parsing to group things together and only falls back on the folder name if filename parsing fails. You should be able to put the vol marker in the middle instead of the end and it should work (i tested it locally and it worked though it dropped the words betrayal and rulebook)

**Comment by Kimbaras** [comment]
> so the problem is that kavita doesnt group things based on folder structure. the folder requirements is there to help the scan loop. PDFs rely rather heavy on filename parsing to group things together and only falls back on the folder name if filename parsing fails. You should be able to put the vol marker in the middle instead of the end and it should work (i tested it locally and it worked though it dropped the words betrayal and rulebook)

Can confirm that trick works.
Regarding the dropped words, if I understood the behaviour correctly, there is no way to rename the file from Kavita UI, correct?

Thanks!

**Comment by majora2007** [comment]
That is correct. I am investigating and working on improving the PDF parser in this v0.8.x series of releases.

**Comment by Kimbaras** [comment]
> That is correct. I am investigating and working on improving the PDF parser in this v0.8.x series of releases.

Thanks for the feedback.

At this point, if you could just reply to the last question at the end of my post (which may be a little off-topic for this issue), it would be great.

After this, from my side you may close this one, or if you prefer to keep this open for the future is up to you!

Thanks again!

**Comment by majora2007** [comment]
> Also, side question: on the docs (https://wiki.kavitareader.com/guides/scanner/managefiles) there is a mention on additionals keywords for "Specials", this is related to the names of the files or additionals sections that can be displayed on Kavita UI?

These are reserved keywords that if found in the filename will cause Kavita to treat it like a special, much like the SP01 marker does. For manga/comics there is a Special tab. For books, unless your books are encoded as Volumes, then you wouldn't see a Special tab. 

For comics explicitly there is a different list of what is considered a special:
https://wiki.kavitareader.com/guides/scanner/comic

**Comment by Kimbaras** [comment]
> These are reserved keywords that if found in the filename will cause Kavita to treat it like a special, much like the SP01 marker does. For manga/comics there is a Special tab. For books, unless your books are encoded as Volumes, then you wouldn't see a Special tab.

Uh then there is something not right here...

Please take a look at the fouth image I've posted, the one reading "Prima Edizione" where you can see the "Special" tab with two books in it.
That libray is marked as "Books" in the config.

Based on your comment, that "Specials" tab shouldn't be there... if I did not misuderstand your comment

**Comment by majora2007** [comment]
Book library works a bit differently. Books can be encoded as either volumes or specials. PDFs will almost always be specials unless they have explicitly the volume keyword. Epubs can be encoded as volumes if they use the series_index tab or belongs to collection tags. 

Hope this helps to clarify.
